panther_analysis_tool/main.py

```diff
@@ -210,6 +210,14 @@ def zip_analysis(args: argparse.Namespace) -> Tuple[int, str]:
     # The colon character is not valid in filenames.
     current_time = datetime.now().isoformat(timespec="seconds").replace(":", "-")
     filename = "panther-analysis-{}.zip".format(current_time)
+    if args.out:
+        if not os.path.isdir(args.out):
+            logging.info(
+                "Creating directory: %s",
+                args.out,
+            )
+            os.mkdir(args.out)
```

Comment:
Does this work recursively to create nested directories? I would guess so just not sure if  you need some `recursive=true` input  or  something.

Reply:
sure doesn't! nice catch... I updated to `os.makedirs()` which can handle nested directories.  Test cases still pass and tested manually